app/utils/filesystem/dialog.ts

```diff
@@ -18,31 +20,23 @@ export const loadDialog = (event, arg) => {
 };
 
 const selectTimeline = (event) => {
-  dialog.showOpenDialog(
-    {
-      title: 'Select a jsPsych timeline file',
-      properties: ['openFile', 'promptToCreate'],
-    },
-    (filePaths) => {
-      if (filePaths) {
-        event.sender.send('loadDialogReply', filePaths[0]);
-      }
-    }
-  );
+  const filePaths = dialog.showOpenDialog({
+    title: 'Select a jsPsych timeline file',
```

Comment:
this should be `lab.js`. does this now support loading an experiment from json?

Reply:
No, unfortunately we've got these jspsych and lab.js systems awkwardly living side by side and importing lab.js files is still not supported.

After Jiffy Lube, there's a big refactor I'd like to do at some point to clear out the old jspsych and make arbitrary lab.js experiments possible